src/tests/index.test.js

```diff
@@ -0,0 +1,549 @@
+describe('Homework - 4', () => {
+    describe('Test init method', () => {
+        describe('ArrayList.init', function (){
+            it('should be defined', function() {
+                assert.isDefined(ArrayList.init, 'method has been defined')
+            });
+            it('should warning user to [] (undefined)', () => {
+                const arr = undefined;
+                const expSize = 'Please input an array';
+                const expArr = 'Please input an array';
+
+                const actual = ArrayList.init(arr);
+
+                assert.deepEqual(actual, expArr);
+                assert.deepEqual(actual, expSize)
+            });
+
+            it('should warning user to [] (null)', () => {
+                const arr = null;
+                const expArr = 'Please input an array';
+                const expSize = 'Please input an array';
```

Comment:
Same mistake. As i see you wrote that program should warn with the empty array but you returning a message.